azkaban-common/src/main/java/azkaban/utils/AzkabanPatternLayout.java

```diff
@@ -0,0 +1,112 @@
+package azkaban.utils;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import org.apache.log4j.helpers.PatternConverter;
+import org.apache.log4j.helpers.PatternParser;
+import org.apache.log4j.PatternLayout;
+import org.apache.log4j.spi.LoggingEvent;
+
+
+/**
```

Comment:
Did you rename this file and class name? It would be best in the future to rename the file first in a separate commit to make the diff easier to track.